shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -390,39 +432,44 @@ export default class PartialClosure extends Component {
               selectedValue={getCurrency}
               onSelect={this.handleSetGetValue}
               label="You buy"
+              tooltip="The amount you receive after the swap"
               disabled
               currencies={currencies}
               usd={getUsd}
             />
             {
-              isSearching && (
+              isSearching || (isNonOffers && maxAmount === 0) && (
```

Comment:
Здесь условие может сработать неверно, так как И приоритетней ИЛИ:
```suggestion
              (isSearching || (isNonOffers && maxAmount === 0)) && (
```